tests/pom.xml

```diff
@@ -842,73 +843,6 @@
                     </wait>
                   </run>
                 </image>
-                <!-- ##### OpenTelemetry collector component, serving as agent, forwarding traces to the jaeger collector ##### -->
```

Comment:
Each property and config file used within this block but not used anywhere else can be removed where it is defined.

Reply:
`opentelemetry-collector.image.name` can also be removed from the Hono bom.